src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.logic.Parser;
+import duke.logic.Storage;
+import duke.task.TaskManager;
+import duke.ui.Ui;
+
+/**
+ * Represents the driver class to run the Duke programme.
+ */
+public class Duke {
+
+    /**
```

Comment:
I don't think you have the type javadocs for every variable in your code. Some times the code is self explanatory. However, please  check with your tutor on this particular feedback (=